runtime/moonbase/tests/common/mod.rs

```diff
@@ -211,6 +211,8 @@ pub const CHARLIE: [u8; 20] = [6u8; 20];
 pub const DAVE: [u8; 20] = [7u8; 20];
 pub const EVM_CONTRACT: [u8; 20] = [8u8; 20];
 
+pub const CROWDLOAN_PAYABLE_PER_BLOCK: u128 = 5_952_380_952_380_952_380;
```

Comment:
This is for a 1_500_000 GLMR user. Maybe we can  add a comment specifying that?

Reply:
I mean a user whose reward is 1_500_000 GLMR

Reply:
we could even calculate it on the integration test but lets leave  that for another PR

Reply:
```suggestion
/// Payable per block for a contributor receiving 1_500_000 * GLMR tokens 
pub const CROWDLOAN_PAYABLE_PER_BLOCK: u128 = 5_952_380_952_380_952_380;
```

Reply:
Can we just do the arithmetic here? (e.g. `pub const CROWDLOAN_PAYABLE_PER_BLOCK: u128 = 1_500_000 * WHATEVER_PER_BLOCK`)